container/mesos/client.go

```diff
@@ -100,7 +100,8 @@ func (self *client) getContainer(id string) (*mContainer, error) {
 	// Check if there is a container with given id and return the container
 	for _, c := range cntrs.Containers {
 		if c.ContainerID.Value == id {
-			return &c, nil
+			container := mContainer(c)
```

Comment:
make doesn't complain any error with fresh checkout of the repo. Not sure why this is failing